charts/puris/Chart.yaml

```diff
@@ -35,7 +35,7 @@ dependencies:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.6.4
+version: 2.6.5
```

Comment:
Please change again to 2.6.4. I finally managed to make the job for helm fail but not block merging for not bumping versions. Also updated future PR templates. See #450